### DISCUSSION TITLE
Welcome to scripts Discussions!

### DISCUSSION THREAD

**Discussion by Dan13615** [Dan13615/scripts#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Dan13615** [comment]
Hello and goodbye

**Reply by H4rdeol** [reply]
salut bébou

**Reply by dragusheen** [reply]
mon chibrax un peu

**Comment by H4rdeol** [comment]
yo les petit potes

**Reply by Dan13615** [reply]
yooooooo

**Comment by dragusheen** [comment]
Coucou bebou !

**Reply by Dan13615** [reply]
Bébouuuuuu

**Reply by H4rdeol** [reply]
UwU
